ui/src/preview/html-preview.js

```diff
@@ -812,7 +817,8 @@ function media(parent, id, xml, options, preload) {
         preload.addFiles(tmpUrl);
         
         self.iframe = $('<video id="' + self.containerName + '-vid" preload="auto" ' + ((self.options["mute"] == 1) ? 'muted' : '') + ' ' + (loop ? 'loop' : '') + '><source src="' + tmpUrl + '">Unsupported Video</video>');
-        
+        self.checkIframeStatus = false;
```

Comment:
Would I be right in saying that we should also set this to false on flash? 

Is it also clearer to set it to false for image/audio even though those don't have an `iframe` set?

Reply:
Yeah, I think we also need to do that :)